src/actions/onboardingActions.js

```diff
@@ -340,11 +336,11 @@ export const finishOnboardingAction = (retry?: boolean) => {
     );
     dispatch(initialDeeplinkExecutedAction());
 
-    logBreadcrumb('onboarding', 'finishOnboardingAction: checking for FEATURE_ONBOARDING flag for enable onboarding');
-    const enableOnboarding = firebaseRemoteConfig.getString(REMOTE_CONFIG.FEATURE_ONBOARDING);
+    logBreadcrumb('onboarding', 'finishOnboardingAction: dispatching getTutorialDataAction');
+    await dispatch(fetchTutorialDataIfNeededAction());
 
     const { onboarding: { tutorialData } } = getState();
-    const routeName = tutorialData && enableOnboarding ? TUTORIAL_FLOW : HOME;
+    const routeName = tutorialData ? TUTORIAL_FLOW : HOME;
```

Comment:
What about case, when we first fetch onboarding data and save it into redux, but afterwards that tutorial data is disabled on the backend. Would suggest keeping the `&& enableOnboarding` part for such case.

Reply:
👍